code/modules/admin/view_variables/debug_variable_appearance.dm

```diff
@@ -0,0 +1,203 @@
+			if(NAMEOF(appearance, color))
+				value = appearance.color
+			if(NAMEOF(appearance, desc))
+				value = appearance.desc
+			if(NAMEOF(appearance, gender))
+				value = appearance.gender
+			if(NAMEOF(appearance, icon))
+				value = appearance.icon
+			if(NAMEOF(appearance, icon_state))
+				value = appearance.icon_state
+			if(NAMEOF(appearance, invisibility))
+				value = appearance.invisibility
+			if(NAMEOF(appearance, infra_luminosity))
+				value = appearance.infra_luminosity
+			if(NAMEOF(appearance, filters))
+				value = appearance.filters
+			if(NAMEOF(appearance, layer))
+				value = appearance.layer
+			if(NAMEOF(appearance, luminosity))
+				value = appearance.luminosity
+			if(NAMEOF(appearance, maptext))
+				value = appearance.maptext
+			if(NAMEOF(appearance, maptext_width))
+				value = appearance.maptext_width
+			if(NAMEOF(appearance, maptext_height))
+				value = appearance.maptext_height
+			if(NAMEOF(appearance, maptext_x))
+				value = appearance.maptext_x
+			if(NAMEOF(appearance, maptext_y))
+				value = appearance.maptext_y
+			if(NAMEOF(appearance, mouse_over_pointer))
+				value = appearance.mouse_over_pointer
+			if(NAMEOF(appearance, mouse_drag_pointer))
+				value = appearance.mouse_drag_pointer
+			if(NAMEOF(appearance, mouse_drop_pointer))
+				value = appearance.mouse_drop_pointer
+			if(NAMEOF(appearance, mouse_drop_zone))
+				value = appearance.mouse_drop_zone
+			if(NAMEOF(appearance, mouse_opacity))
+				value = appearance.mouse_opacity
+			if(NAMEOF(appearance, name))
+				value = appearance.name
+			if(NAMEOF(appearance, opacity))
+				value = appearance.opacity
+			if(NAMEOF(appearance, overlays))
+				value = appearance.overlays
+			if("override")
+				var/image/image_appearance = appearance
+				value = image_appearance.override
+			if(NAMEOF(appearance, pixel_x))
+				value = appearance.pixel_x
+			if(NAMEOF(appearance, pixel_y))
+				value = appearance.pixel_y
+			if(NAMEOF(appearance, pixel_w))
+				value = appearance.pixel_w
+			if(NAMEOF(appearance, pixel_z))
+				value = appearance.pixel_z
+			if(NAMEOF(appearance, plane))
+				value = appearance.plane
+			if(NAMEOF(appearance, render_source))
+				value = appearance.render_source
+			if(NAMEOF(appearance, render_target))
+				value = appearance.render_target
+			if(NAMEOF(appearance, suffix))
+				value = appearance.suffix
+			if(NAMEOF(appearance, text))
+				value = appearance.text
+			if(NAMEOF(appearance, transform))
+				value = appearance.transform
+			if(NAMEOF(appearance, underlays))
+				value = appearance.underlays
+
+			if(NAMEOF(appearance, parent_type))
+				value = appearance.parent_type
+			if(NAMEOF(appearance, type))
+				value = "/appearance (as [appearance.type])" // don't fool people
+
+			// These are not documented ones but trackable values. Maybe we'd want these.
+			if(NAMEOF(appearance, animate_movement))
+				value = appearance.animate_movement
+			if(NAMEOF(appearance, dir))
+				value = appearance.dir
```

Comment:
this one is inherited from `/atom` but we can check this in /appearance

Reply:
![image](https://github.com/tgstation/tgstation/assets/87972842/f0b803ee-29d9-4554-9d67-a58ada39365d)
![image](https://github.com/tgstation/tgstation/assets/87972842/6561d5e5-dcf1-4567-8513-cdf0165faadd)

Reply:
alright